templates/jlink/jlink-builder/README.md

```diff
@@ -0,0 +1,56 @@
+# OpenShift-JLink PoC
+
+Try it out:
+
+## Stage 1: create ubi9 jlink imagestream and push a ubi9 image with jmods installed to it.
+
+You need:
+
+1. Access to an OpenShift instance, such as crc
+
+Steps to produce the imagestream and image:
+
+1. Switch to the openshift project and install the template
```

Comment:
remove `switch to the openshift project and`: It's normal to use that namespace but not essential. OOTB with CRC for example the developer user cannot write to it. I'm following along here entirely in a separate project.